docs/blueprints/EP031.rst

```diff
@@ -280,10 +278,10 @@ VII. New EVC attribute
 
 The **telemetry_int** napp will leverage the EVC's metadata attribute to create a new item, called `telemetry`. This new item will be a dictionary will the following values:
 
-  * "enabled": [True|False]
-  * "source": dpid/name of the switch to be used as the INT Source switch (Future use).
-  * "sink": dpid/name of the switch to be used as the INT Sink switch (Future use).
-  * "last_enabled": timestamp of when the item "enabled" changed. 0 for never.
+  * "status": "enabled|disabled"
+  * "status_reason": ["some_error"]
```

Comment:
Since we have certain events or asynchronous side effects that can happen in addition to a HTTP request from clients, then we can use `"status_reason"` here too to fill out any error (they'll be specified as we go, but basically they'll be enum string values).